pkg/apis/core/types.go

```diff
@@ -2848,7 +2848,7 @@ type PodSpec struct {
 	// The RuntimeClass admission controller will reject Pod create requests which have the overhead already
 	// set. If RuntimeClass is configured and selected in the PodSpec, Overhead will be set to the value
 	// defined in the corresponding RuntimeClass, otherwise it will remain unset and treated as zero.
-	// More info: https://git.k8s.io/enhancements/keps/sig-node/20190226-pod-overhead.md
+	// More info: https://git.k8s.io/enhancements/keps/sig-node/688-pod-overhead/README.md
 	// This field is alpha-level as of Kubernetes v1.16, and is only honored by servers that enable the PodOverhead feature.
```

Comment:
thanks for fixing this. while we're here, the field is now beta-level as of 1.18 (applies everywhere)

Reply:
Thanks for your reminding. I will update the level description of this field